src/frontend/scripts/src/tools/appendToFile.ts

```diff
@@ -0,0 +1,39 @@
+import fs from "fs";
+import path from "path";
+import searchInFile from "./searchInFile";
+
+export default function appendToFile(
+  filePath: string,
+  line: string,
+  stringToFind?: string,
+  after?: boolean
+) {
+  try {
+    if (!stringToFind) {
+      fs.appendFileSync(filePath, line);
+      console.log(`Appended line in ${path.basename(filePath)}`);
+      return;
+    }
+
+    var data = fs.readFileSync(filePath).toString().split("\n");
```

Comment:
```suggestion
    const data = fs.readFileSync(filePath).toString().split("\n");
```
There are others `var` in this file and others, please use `const` or `let` instead.